src/rust/ensogl/src/display/shape/text/glyph/msdf.rs

```diff
@@ -2,63 +2,73 @@
 
 use crate::prelude::*;
 
-use ensogl_core_msdf_sys as msdf_sys;
-use msdf_sys::MultichannelSignedDistanceField;
-use nalgebra::clamp;
+use ensogl_core_msdf_sys::Msdf;
+
+
 
 // ====================
 // === MSDF Texture ===
 // ====================
 
-/// Texture with msdf for all loaded glyph of font
+/// MSDF texture data for all loaded glyph of a font.
 ///
-/// This structure keeps texture data in 8-bit-per-channel RGB format, which
-/// is ready to be passed to webgl texImage2D. The texture contains MSDFs for
-/// all loaded glyph, organized in vertical column.
-#[derive(Debug,Default)]
-pub struct MsdfTexture {
+/// This structure keeps texture data in 8-bit-per-channel RGB format, which is ready to be passed
+/// to WebGL `texImage2D`. The texture contains MSDFs for all loaded glyph, organized in vertical
+/// column.
+#[derive(Clone,CloneRef,Debug,Default)]
```

Comment:
As we've been chatting via phone, lets leave it because `Rc` is implementation detail of this structure. In case someone would like to have non clone-refable struct, they are free to add it.